godot-codegen/src/generator/enums.rs

```diff
@@ -127,12 +148,27 @@ pub fn make_enum_definition_with(
 
             impl crate::meta::FromGodot for #name {
                 fn try_from_godot(via: Self::Via) -> std::result::Result<Self, crate::meta::error::ConvertError> {
-                    // Pass i32/u64 enum/bitfield as i64 on the FFI layer. Only necessary for bitfields (u64).
-                    // Bitfields are cast to i64 for FFI, then reinterpreted in C++ as uint64_t.
                     <Self as #engine_trait>::try_from_ord(via)
-                        .ok_or_else(|| crate::meta::error::FromGodotError::InvalidEnum.into_error(via as i64))
+                        .ok_or_else(|| crate::meta::error::FromGodotError::InvalidEnum.into_error(via))
                 }
             }
+
+  impl crate::registry::property::Var for #name {
+      fn get_property(&self) -> Self::Via{
+          <Self as #engine_trait>::ord(*self)
+      }
+
+      #var_trait_set_property
+
+      fn var_hint() -> crate::meta::PropertyHintInfo{
+          crate::meta::PropertyHintInfo{
+              hint: #property_hint,
+              hint_string: #enum_hint_string.into(),
```

Comment:
Please use explicit `GString::from()` here.
```suggestion
              hint_string: GString::from(#enum_hint_string),
```